practices/front-end-ios.md

```diff
@@ -0,0 +1,82 @@
+
+- **@ios-front-enders** Engineers who work on Artsy's iOS codebases in a native capacity.
+- **@app-release-team** The
+  [iOS App Quality Team 🔒](https://www.notion.so/artsy/Artsy-iOS-App-Quality-Team-58b9b55f52484000b9689edd809e7016)
+  is a focused team of one Designer, one Product Manager, and one Engineer who help ensure the
+  [Artsy iOS app](http://iphone.artsy.net)'s quality.
+
+The Front-End iOS Practice meetings every two weeks; meeting notes are
+[available in Notion](https://www.notion.so/artsy/Front-End-iOS-ecc07763bfd04a848c74107dde3ec6dc).
+
+### Team Goals
+
+Our goals are the same as the [Front-End Practice goals](./front-end.md#team-goals), but with a focus on supporting
+iOS software at Artsy. That means we have the following additional goals:
+
+- Build tools and systems to help non-iOS engineers contribute to Artsy iOS software using React Native.
+- Help our colleagues understand the differences between building iOS software and web software.
+
+### Logistics
+
+Each iOS app at Artsy is deployed according to its own unique needs. See their repos (listed
+[below](#artsy-ios-apps)) for deployment instructions.
+
+### Our Technologies
+
+Initially, Artsy apps were built with Objective-C. Then Swift was announced in 2014, and
+[we tried it](http://artsy.github.io/blog/2017/02/05/Retrospective-Swift-at-Artsy/) but have ultimately
+[settled on React Native](http://artsy.github.io/blog/2018/03/17/two-years-of-react-native/). We will still use
+Objective-C and Swift for the foreseeable future, and native code will likely always make sense for parts of our
+app (notably, [the Augmented Reality feature](http://artsy.github.io/blog/2018/03/18/ar/) and the animation-driven
+[Live Auctions Integration](http://artsy.github.io/blog/2016/08/09/the-tech-behind-live-auction-integration/#The.iOS.native.app:.Eigen),
+user interface). New features are being built using React Native; the remaining native parts of our main app are
+[listed here 🔒](https://www.notion.so/artsy/Eigen-migration-to-React-Native-54dda83b023b4cb4965a8defdae9687f)
+alongside decisions and priorities for switching them to React Native. We have no plans at this time to rewrite our
+native-only projects in React Native.
+
+### Artsy iOS Apps
+
+Artsy has the following iOS applications, which are all open source.
+
+- [Eigen](https://github.com/artsy/eigen) is Artsy's main iOS app. (Objective-C, Swift)
+  - [Emission](https://github.com/artsy/emission) is this app's React Native component library (see
+    [this post](http://artsy.github.io/blog/2018/04/17/making-a-components-pod/)). (TypeScript)
+- [Energy](https://github.com/artsy/energy) is Artsy's partner app. (Objective-C)
+- [Eidolon](https://github.com/artsy/eidolon) is Artsy's auctions kiosk. (Swift)
+- [Emergence](https://github.com/artsy/emergence) is Artsy's Apple TV app. (Swift)
+
+All Artsy iOS codebases start with the letter `e`, but not all Artsy codebases that start with the letter `e` are
```

Comment:
Oh, that's fascinating. I wasn't aware of this! Now that I think about it though...